src/Core/Models/Domain/Send.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Bit.Core.Abstractions;
+using Bit.Core.Enums;
+using Bit.Core.Models.Data;
+using Bit.Core.Models.View;
+using Bit.Core.Utilities;
+
+namespace Bit.Core.Models.Domain
+{
+    public class Send : Domain
+    {
+        public string Id { get; set; }
+        public string AccessId { get; set; }
+        public string UserId { get; set; }
+        public SendType Type { get; set; }
+        public CipherString Name { get; set; }
+        public CipherString Notes { get; set; }
+        public SendFile File { get; set; }
+        public SendText Text { get; set; }
+        public CipherString Key { get; set; }
+        public int? MaxAccessCount { get; set; }
+        public int AccessCount { get; set; }
+        public DateTime RevisionDate { get; set; }
+        public DateTime? ExpirationDate { get; set; }
+        public DateTime DeletionDate { get; set; }
+        public string Password { get; set; }
+        public bool Disabled { get; set; }
+
+        public Send() { }
```

Comment:
Would this also need to call the `base()` constructor? and perhaps the loaded constructor calls `this()`